lib/fastlane/plugin/wpmreleasetoolkit/actions/common/upload_to_s3.rb

```diff
@@ -0,0 +1,107 @@
+require 'fastlane/action'
+require 'digest/sha1'
+
+module Fastlane
+  module Actions
+    module SharedValues
+      S3_UPLOADED_FILE_PATH = :S3_UPLOADED_FILE_PATH
+    end
+
+    class UploadToS3Action < Action
+      def self.run(params)
+        file_path = params[:file]
+        file_name = File.basename(file_path)
+        UI.user_error!("Unable to read file at #{file_path}") unless File.file?(file_path)
+
+        bucket = params[:bucket]
+        key = params[:key]
+
+        if params[:auto_prefix] == true
+          file_name_hash = Digest::SHA1.base64digest(file_name)
```

Comment:
Note: the Base64 alphabet happens to contain the `/` character. Will that risk causing an issue or result in misleading or strange cases?

For example if the input file is named `input_file_1` then the base64 digest of its SHA1 is `k5w5OY2yQF55HiBXeP9w+F3/Yg4=` with a `/` in the middle. Worse, I imagine there could be input file names that lead to a base64 SHA1 to _end with_ a `/`, which would lead to a key containing two consecutive `/` once we assemble that prefix with the filename, and I can imagine a key with `foo//bar` to lead to errors with S3…

Discovered that use case by implementing Unit Tests for this in #340 btw… so they _are_ really useful (in case that was still needed to be proven) 😛 

I suggest we should use `hexdigest` instead of `base64digest` to avoid such situations:
```suggestion
          file_name_hash = Digest::SHA1.hexdigest(file_name)
```